verilog/tools/lint/BUILD

```diff
@@ -48,6 +48,7 @@ _linter_test_configs = [
     ("parameter-type-name-style", "parameter_type_name_style", False),
     ("parameter-type-name-style", "localparam_type_name_style", False),
     ("plusarg-assignment", "plusarg_assignment", True),
+    ("port-name-suffix", "port_name_suffix", False),
```

Comment:
can you also add a commented-out entry to https://cs.opensource.google/verible/verible/+/master:verilog/analysis/default_rules.h ?